packages/azure-themes/src/azure/AzureThemeLight.ts

```diff
@@ -5,55 +5,91 @@ import { FontSizes } from './AzureType';
   bodyBackground: LightSemanticColors.background,
   bodyText: LightSemanticColors.text.body,
   bodyDivider: CommonSemanticColors.dividers.lineSeparator,
   buttonBackground: LightSemanticColors.secondaryButton.rest.background,
   buttonBackgroundChecked: LightSemanticColors.secondaryButton.pressed.background,
   buttonBackgroundCheckedHovered: LightSemanticColors.secondaryButton.hover.background,
-  buttonBackgroundDisabled: LightSemanticColors.disabledButton.background,
+  buttonBackgroundDisabled: '#F3F2F1',
   buttonBackgroundHovered: LightSemanticColors.secondaryButton.hover.background,
-  buttonBackgroundPressed: LightSemanticColors.secondaryButton.pressed.background,
-  buttonText: LightSemanticColors.secondaryButton.rest.border,
+  buttonBackgroundHoveredDefault: '#F3F2F1',
+  buttonBackgroundPressed: '#EDEBE9',
+  buttonBorderColor: '#8A8886',
+  buttonBorderHovered: '#8A8886',
+  buttonText: '#323130',
   buttonTextChecked: LightSemanticColors.secondaryButton.pressed.border,
   buttonTextCheckedHovered: LightSemanticColors.secondaryButton.hover.border,
-  buttonTextDisabled: LightSemanticColors.disabledButton.text,
+  buttonTextDisabled: '#A19F9D',
   buttonTextHovered: LightSemanticColors.secondaryButton.hover.border,
   buttonTextPressed: LightSemanticColors.secondaryButton.pressed.border,
+  checkboxBorder: '#323130',
+  checkboxBorderHover: '',
+  checkboxBorderChecked: '',
+  checkmarkColorHover: '#605E5C',
+  checkboxBorder2: '#323130',
+  checkboxBorderDisabled: '#C8C6C4',
+  checkboxCheckedDisabled: '#C8C6C4',
+  checkboxCheckedMarkDisabled: '#ffffff',
+  checkboxIndeterminateBackground: '#ffffff',
+  checkMarkFill: '#ffffff',
+  controlGroupBorder: '#323130',
+  checkBoxselectedBorderHover: '#005A9E;',
+  groupChoiceFill: '#0078D4',
+  contextMenuBackground: 'transparent',
   disabledBackground: CommonSemanticColors.backgrounds.disabled,
-  disabledBodyText: LightSemanticColors.text.disabled,
+  disabledToggleBackground: 'transparent',
+  disabledBodyText: '#A19F9D',
+  disabledIconColor: 'rgb(200, 198, 196)',
+  dropdownBorder: '#8A8886',
+  dropdownBorderHover: '#323130',
   errorBackground: LightSemanticColors.controlOutlines.error,
   errorText: LightSemanticColors.text.error,
-  focusBorder: LightSemanticColors.controlOutlines.accent,
+  focusBorder: '#0078D4',
+  iconButtonColor: '#0078D4',
+  iconButtonDisabledColor: '#A19F9D',
+  iconHoveredColor: '#106EBE',
+  iconPressedColor: '#005A9E',
   inputBackground: LightSemanticColors.background,
-  inputText: LightSemanticColors.text.value,
+  inputText: '#323130', //LightSemanticColors.text.value,
   inputBorder: CommonSemanticColors.textControlOutline.rest,
   inputBorderHovered: CommonSemanticColors.textControlOutline.hover,
-  inputPlaceholderText: LightSemanticColors.text.placeholder,
-  link: LightSemanticColors.text.hyperlink,
-  linkHovered: LightSemanticColors.text.hyperlink,
+  inputPlaceholderText: '#323130', //LightSemanticColors.text.placeholder,
```

Comment:
remove this commented code if it's unnecessary.

Reply:
Complete